### ISSUE TITLE
Coerce bool to int

### ISSUE THREAD

**Issue by mythsmith** [QQuick/Transcrypt#391]
If you try this:
```
console.log('PRE')
a = int(True)
console.log('POST', a)
```
It will result in: 
```
PRE
uncaught exception: unknown (can't convert to string)
```
And never prints POST. Enabling __pragma__('tconv') does not help. 
As a side node, it seems this was not a problem before updating from 3.6.24 to 3.6.50.

**Comment by JdeH** [comment]
Doesn't look right...
Will see what can be done about it.

Kind regards
Jacques

**Comment by JdeH** [comment]
Will be in next commit.

**Comment by JdeH** [comment]
Committed and available via PyPi  in v3.6.53

**Comment by mythsmith** [comment]
Thanks!
